audit/gdcd/GetProductSubProduct.go

```diff
@@ -24,7 +30,7 @@ func GetProductSubProduct(project string, page string) (string, string) {
 		"docs":                     "Server",
 		"docs-k8s-operator":        "Enterprise Kubernetes Operator",
 		"docs-relational-migrator": "Relational Migrator",
-		"entity-framework":         "Drivers", // Missing from taxonomy/this is a guess
+		"entity-framework":         "Entity Framework Core Provider", // DOCSP-50997 to add to taxonomy
```

Comment:
So for this one, if we're changing the name, we _also_ need to run DODEC and change the name in all the existing MongoDB documents in the collection. That's this bad boy: https://github.com/mongodb/code-example-tooling/blob/073f87ce5eb87660601e4915420b6111d1f475cd/audit/dodec/src/updates/ChangeProductName.go

I consider this a prereq for merging this update, because I don't want to mix product names/counts in the reports.